

message EdgeList {
    repeated uint64 nbrs = 1;
}

